docs/building.md

```diff
@@ -38,10 +38,56 @@ ctest -L performance
    |    TEST\_INTEL\_OPTIMIZATION | OFF      | Option | Test intel optimization and vectorization |
    |    CHECK\_REGISTRY\_PRESSURE | OFF      | Option | Check the registry pressure for Kokkos CUDA kernels |
    |               BUILD\_TESTING | ON       | Option | Multi-testing enablement |
-   | PARTHENON\_DISABLE\_EXAMPLES | OFF      | Option | Toggle building of examples, if regression tests are on, drivers needed by the tests will still be built |   
+   | PARTHENON\_DISABLE\_EXAMPLES | OFF      | Option | Toggle building of examples, if regression tests are on, drivers needed by the tests will still be built |
    |   ENABLE\_INTEGRATION\_TESTS | ${BUILD\_TESTING} | Option | Enable integration tests |
    |    ENABLE\_REGRESSION\_TESTS | ${BUILD\_TESTING} | Option | Enable regression tests |
    |          ENABLE\_UNIT\_TESTS | ${BUILD\_TESTING} | Option | Enable unit tests |
    |               CODE\_COVERAGE | OFF      | Option | Builds with code coverage flags |
+   |       CMAKE\_INSTALL\_PREFIX | machine specific | String | Optional path for shared library installation |
 
 ### NB: CMake options prefixed with *PARTHENON\_* modify behavior.
+
+## Building a shared library
+
+An alternative to building Parthenon alongside a custom app (as in the examples)
+is to first build Parthenon separately as a shared library and then link to it
+in the app.
+
+To build Parthenon as a shared library, provide a *CMAKE\_INSTALL\_PREFIX* path
+to the desired install location to the Parthenon cmake call, then call *make install*:
+
+```bash
+cmake -DCMAKE_INSTALL_PREFIX="$your_install_dir" $parthenon_source_dir
+make -j install
```

Comment:
Recommend `cmake --install .` instead since it's generator-agnostic.

Reply:
Thanks, didn't know about this. Now:
```bash                                                                   
cmake -DCMAKE_INSTALL_PREFIX="$your_install_dir" $parthenon_source_dir           
cmake --build "$your_build_dir" -- -j                                            
cmake --install "$your_build_dir"         
```

Reply:
`cmake ---build . --parallel` should work as well. And `cmake -DCMAKE_INSTALL_PREFIX="$your_install_dir" -S $parthenon_source_dir -B .` would be even more modern.

Reply:
Added `--parallel`, thanks @junghans (keeping the separate configure/build calls now just because they're more explicit)